% setup
\documentclass[11pt]{article}
\usepackage[style=ieee,sorting=none]{biblatex}
\usepackage{etoolbox}
\usepackage{fancyhdr} % For custom headers/footers
\usepackage[a4paper, left=0.5in, right=0.5in, top=0.5in, bottom=0.5in, headheight=14pt, headsep=10pt, footskip=14pt]{geometry}
\usepackage{graphicx}
\usepackage[colorlinks=true, linkcolor=blue, urlcolor=blue, citecolor=blue]{hyperref}
\usepackage{mathptmx} % Use Times New Roman font

% title info
\title{CS 698R Literature Review}
\author{Wesley Borden}
\date{July 8, 2025}

% header/footer
\pagestyle{fancy}
\fancyhf{} % Clear default header and footer
\fancyhead[L]{CS 698R Literature Review} % Left header
\fancyhead[R]{Wesley Borden} % Right header
\fancyfoot[C]{\thepage} % Center footer with page number
\renewcommand{\headrulewidth}{0pt}

% spacing and indentation
\AtBeginEnvironment{document}{\setlength{\parindent}{0.25in}} % Ensure parindent is set for the document
\newcommand{\sectionwithindent}[1]{
    \section*{#1}
    \hspace{\parindent} % Indent the first paragraph
}
\newcommand{\subsectionwithindent}[1]{
    \subsection*{#1}
    \hspace{\parindent} % Indent the first paragraph
}

% other package setup
\addbibresource{review.bib}

% text
\begin{document}

\sectionwithindent{Data Science is Central to Ongoing Advancements in Neuroscience}
% Neuroscience needs data science
In 2018, \textit{Nature Methods} published an article stating that ``Neuroscience is experiencing a revolution'' \cite{pandarinath2018autoencoders}. The article introduced a novel computational approach, implementing a neural network-based model to infer functional relationships between active brain cells. As developments in neuroscience continue to unfold, that article is one of many suggesting that computing, automation, data analysis, and machine learning will increasingly be at the core of major research achievements and clinical applications in neuroscience.

% Data science has a big opportunity in biology--specifically neuroscience
Since the design of the transformer architecture in 2017 \cite{vaswani2023attentionneed} and commercialization of scaled machine learning in recent years, large language models (LLMs) have rapidly become a ubiquitous technology \cite{naveed2025llm}. Though much public attention and industry effort has focused on LLMs and other consumer-facing tools, some of the greatest achievements in machine learning are outside the scope of these applications. For example, machine learning applications have made major contributions to computational biology. In that domain, the AlphaFold algorithms have implemented a model similar to the transformer to increase the number of all known protein structures in the world from less than a few hundred thousand to hundreds of millions \cite{jumper2021alphafold, varadi2024alphafolddb}. Just as some of the next decade's greatest achievements in biology and neuroscience will require applications of data science and machine learning, some of the greatest opportunities to innovate with data science and machine learning lie in applications to other sciences, including biology and neuroscience.

% Network science is a specific part of data science that is key for neuroscience and connectomics.
Artificial neural networks (ANNs) are not the only technology showing promise for innovative applications to neuroscience. As the nervous system contains a network of connected, interacting neurons, known as a connectome, network science is similarly applicable. Network science provides a mathematical foundation for modeling connectomes \cite{emmons2015connectomics, easley2010networks, newman2010networks}, including metrics (e.g., centrality, modularity) and algorithms (e.g., page rank and Louvain) for understanding individual nodes (neurons) and their communities within the network \cite{easley2010networks, newman2010networks}. The foundation of network science is combined with ANNs in graph neural networks (GNNs), including graph autoencoders (GAEs), which use convolutions on a product of the graph's adjacency matrix, degree matrix, and node feature embeddings to establish structure-aware embeddings of each node \cite{velickovic2018graphattentionnetworks, kipf2016gae}. As modern methodological advances increase capacity for collection of high-resolution connectomes, these network science tools are already beginning to be applied to improve understanding of nervous tissue function (e.g., \cite{srinivasan2025gnnconnectome, neudorf2022gnnconnectome}).

% The plan for this paper is to show synergy between data science and neuroscience subfields
This review considers the intersection of subfields in data science and neuroscience, including connectomics, extracellular electrophysiology, brain computer interfaces, neural signal processing, and spike train analysis. In introducing these subfields, I suggest that they have significant potential for synergy in accelerating understanding of microscale processing and distributed systems in the brain. In addition to providing understanding, I propose that these same technologies will offer significant new clinical interventions for a wide variety of nervous system diseases and disorders in coming decades.

\subsectionwithindent{A Connectome Details Neural Connections}
% Define a connectome
The term "connectome" refers to the set of all neural connections in the nervous system, originating in the context of post-human-genome-project bioinformatics -omics research \cite{ciarrusta2023connectome, mahapatra2010omics, green2015hgp}. The term "biological neural network" (BNN) \cite{yamazaki2022spiking} is also used to reference neural connections, though a BNN may be only a sub-network of the full network--a partial connectome. The field of connectomics is based on a foundational premise that the aggregated structure and function of individual neural connections determines intermediate- and high-level functionality of the brain, such as cognitive, behavioral, and neurological outcomes. These aggregated neural connections can be considered as a network, with neurons as nodes and relationships between nodes as edges. Relationships are directional and may be excitatory, inhibitory, or modulatory, with relationship strength varying with the quantity of neurotransmitter released from one cell and the expression of neurotransmitter receptors on the other. Many factors at systemic and subcellular scales regulate the network as it changes over time \cite{ciarrusta2023connectome}. Though a full connectome ideally includes all neurons and all synaptic relationships, connectomes are typically studied at varied scales. A partial connectome includes a sub-network of a complete microscale connectome; a macroscale connectome is a connectome without cellular resolution, resulting in analogs of a network science meta-node and some missing nodes or edges; and a functional connectome is based on functional relationships rather than known physical connections between cells \cite{baxter2023functionalconnectome, blommaert2023structuralconnectome, ciarrusta2023connectome, sejnowski2016nanoconnectomics, elam2021hcp}.

% Microscopy
The gold standard for collection of a connectome is ex-vivo electron microscopy, a method that began to be used in the 1980's \cite{white1986structure, emmons2015connectomics}. The process includes delicate slicing of the tissue and imaging of each slice, followed by stitching visualizations together to identify all neurons and connections. Current methods restrict the scale of this exercise to only small animal models such as C. elegans and drosophila melanogaster \cite{white1986structure, emmons2015connectomics, scheffer2020connectome}. In larger mammals, ex-vivo microscopy has been used to produce partial connectomes from samples of neural tissue, and to produce macroscale connectomes from the full brain \cite{motta2019connectomicreconstruction, helmstaedter2011high, amunts2013bigbrain}. Continued methodological advances will likely continue to expand the scope and scale of these processes. While these developments are promising, microscopy is limited to only ex-vivo research, therefore microscopy-based connectomics must be combined with in-vivo neural recording to understand functional relationships and work toward clinical interventions.

% MRI and the human connectome project
A major milestone in connectomics was the Human Connectome Project (HCP) \cite{elam2021hcp}, which pioneered methods for collection of human connectomes in-vivo via magnetic resonance imaging (MRI), including functional MRI (fMRI) and diffusion MRI (dMRI). From 2010 to 2019, teams collected a large initial dataset of fMRI-based macroscale connectomes and standardized many processes for large-scale study with MRI. Many projects today continue to iterate and build on the HCP with expanded scope, considering additional populations and analyses. While these methods have proven extremely useful in many research cases, they observe neural activity only indirectly by measuring fluid flow through tissue. Measurement resolution is also restricted by the physical limitations of radiowaves used in MRI sensors. As a result, cellular-level activity cannot be measured, and determination of causative relationships between nuclei is limited. For example, excitatory vs. inhibitory relationships typically cannot be identified \cite{hagman2010mriconnectomics}.

\subsectionwithindent{Extracellular Electrophysiology is Fundamental to Brain Computer Interfaces}
% Objectives for new methods in connectomics
Recent advances in electrical engineering and data analysis for in-vivo neural recording adds complementary value to the contributions being made by current microscopy and MRI methods. Neural recording enables more specific identification of functional relationships between cells for a given human or animal during a given period in time, which will likely improve understanding of cellular underpinnings of high-level function of the nervous system, and build toward improved clinical interventions for many neurological and psychiatric disorders \cite{kobayashi2025connections, zhang2025neds,musk2019integrated, card2024neuroprosthesis}. Important considerations with these advances in neural recording include: (1) recording should have sufficient resolution in space and time to differentiate precise firing activity of individual cells; (2) though whole-brain-recording is unlikely, a maximal spatial area should be targeted, to capture activity from many neurons, including those in different anatomical regions \cite{jun2017probes}; (3) the connections determined by analysis of the recordings will give functional, though not necessarily anatomical, relationships \cite{ciarrusta2023connectome}; and (4) collected data must be organized to enable efficient, scalable processing and to automate recognition of functional connectomes \cite{pachitariu2016kilosort, buccino2020spikeinterface}.

% Intracranial, extracellular electrophysiology recording
Neural recording with a single electrode and a small sample of neural tissue has been used for nearly a century to understand the nervous system \cite{piccolino1997galvani, adrian1954olfaction}. Before the 21\textsuperscript{st} century 

% 1950's electrodes:
% - green1958microelectrode: https://www.nature.com/articles/182962a0
% - dowben1953electrode: A Metal-Filled Microelectrode: Robert M. Dowben, Jerzy E. Rose

% tetrode: 
% - jog2002tetrode:  Tetrode technology: advances in implantable hardware, neuroimaging, and data analysis techniques (M.S. Jog)
% - wilson1993ensemble: Dynamics of the Hippocampal Ensemble Code for Space (Matthew A. Wilson and Bruce L. McNaughton)

    %%% The oldest method for neural recording uses an electrode placed physically near one or more neurons to measure its electrical voltage directly  [CITE: TODO, pachitariu 2016]. This has often been done with just a few cells on a slide under a microscope  [CITE: TODO], and has also been done in-vivo in animal and human studies [CITE: TODO, braingate, ibl2022datarelease]. Before 20\_\_, we had tetrodes and similar rough probes [CITE: TODO]. They could have strong resolution, but poor coverage (NP paper). These probes have continued to develop, with a modern neuropixels probe containing 9\_\_ sites that can be programmed, up to 38\_ working simultaneously. This allows greater spatial coverage, but multiple probes can also be used together.

    %%% This multipleWith the NP probes, now we get strong coverage too. And that's talking about animal research devices. In-clinic, the early 2000 showed the utah array (discussed earlier, cite), but it was hard on the patients because of a rigid array. (electrode arrays). flexible, deep devices, delicately inserted show a lot of potential for maximizing spatial coverage AND spatiotemporal resolution.

    %%% Talk about BCI's, BrainGate, the Utah Array, Neuralink, Flexible/robotically-inserted fibers

    %%% % Before we can get into the algorithms, we need to understand the data that we are working with, and how it is collected, as well as potential outputs that are algorithms should have. That starts with an understanding of the hardware that is implanted in the nervous tissue. In the late 2000s the brain gate contortion made significant progress with Braden computer interfaces by implanting devices into the brains of multiple research participants. These individuals were severely paralyzed, and we are implanted with a Utah ray. A Utah ray is in a ray electrodes. The array was implanted like a stamp into their motor cortex, the part of their brain that directly controls muscles like the arms or hands. After being given time to heal, the patients were given control of the computer. Or the device was connected to a computer, such that patient could control the cursor as if they were controlling their hand. Now we want to talk about the algorithms that are between hardware and software. But that is what the output should look like. The brain gate consortium has continued to make progress, for example, recently they were able to do full whole processing, rather than requiring a patient to move a cruiser on the screen the patient could think of a specific words, and the words would appear on the screen or something like that. Simultaneous with the brain gay research has been significant advancement at. The device is implanted with a surgical robot and uses much finer and more flexible, metal strands based on prior imaging of the brain with MRI and CT. The Reva is able to maneuver around blood vessels and other delicate tissue in order to precisely place. The devices electrodes at a higher resolution. This is significantly less invasive for the patient, while maintaining greater penetration or coverage or resolution of the brain tissue. In the early device, about seven patients of received the device to date as well as comprehensive animal studies. Uniquely, the NeuroLink device has built its machine learning models directly based off the voltage data, rather than space Spike sorted data.

% This isn't as new as it seems

    %%% , VNS & RNS, DBS & TMS, applications to paralysis, epilepsy, movement disorders, neurodegeneration (ALS), and psychiatric illness (depression-TMS; schizophrenia-DBS)

    %%% % There are other devices that are not typically called brain computer interface, but are doing the same thing. For epilepsy, vagus nerve stimulators are electrical devices that stimulate the vagus nerve, a cranial nerve just outside the brain. It is not a closed loop system, and there is no reading from the brain, but repeated electrical stimulation is given to the third. Nerve. Alter, for treatment, resistant epilepsy, a recurrent nerve stimulator, check the acronym, RNS, reads and writes from a seizure focus, detecting when seizure activity is likely to begin and intervening with inhibitory electrical activity as needed.

    %%% Deep brain stimulation involves open or closed loop stimulation, typically of the basal nuclei. And electrode is inserted surgically, and then controlled from an external device to give repeated simulation or something like that. That and look it up. Site that and look it up. Transcranial magnetic stimulation, involves it device from outside the brain, but uses electromagnetic stimulation to change the brain activity. It has been found effective as a dream at four, severe depression.

    % So we have all of these devices and their developing and really in their infancy and say it in doctors are starting to see applications for movement disorders and neural degeneration and psychiatric illness, and the list goes on.

% citations: ibl2022datarelease, ibl2022iblsorter, jun2017probes, paulk2022probes, steinmetz2021probes
% citations: card2024neuroprosthesis, geller2018rns, heck2014rns, musk2019integrated, vilela2020bci pachitariu2024kilosort4

\subsectionwithindent{Scaled Electrophysiology Relies on Strong Neural Signal Processing} % NOTE: [CITE: TODO, notes] is how outstanding citation plans will work.
% Define the problem neural signal processing faces
The primary output of electrophysiological recording is a set of voltages, each associated with a time and a recording site (channel). These values are typically organized into a matrix, with rows corresponding to channels and columns corresponding to timepoints. With hundreds of channels and thousands of timepoints per second (devices typically work at 1,000-10,000Hz), these matrices can become quite large. This raw data can be used directly in some cases, but it does not provide a set of identified neurons, nor information about the neurons, such as when they fired an action potential. The process of converting the raw voltage data into this more useful information is known as spike sorting, a type of neural signal processing [CITE: TODO, notes]. There are many spike sorting algorithms [CITE: TODO, notes], but we will focus on one algorithm that leads in the field of spike sorting for large sets of parallel-collected signals or channels called KiloSort [CITE: TODO, kilosort 2016 and 2022].

% Detail the KiloSort pipeline
A key initial step in neural signal processing for algorithms like kilosort is to separate the signals by

% Overview usage and impact of spike sorting pipelines




\subsectionwithindent{Processing Spike Trains Can Produce a Functional Connectome} % NOTE: [CITE: TODO, notes] is how outstanding citation plans will work.
% tons of algorithms exist (cite a bunch. Ranging from wisconsin-madison binning to autoencoders to GLM.). Focus on each of:
% Pairwise Statistics: cross-correlation, transfer entropy, granger causality, GLM
% Also Pairwise, but trained: Autoencoder methods (autoLFADS)
% other new methods I won't replicate: CLDS, NEDS
% citations: pandarinath2018autoencoders, zhang2025neds


\subsectionwithindent{Conclusion} % NOTE: [CITE: TODO, notes] is how outstanding citation plans will work.
% breifly mention that we're done talking through details. The big picture is that 20 years from now I anticipate a revolution in medical devices and understanding of neural circutry to be beyond stopping, and similar in scope and size and hype to the modern AI boom. And it will all be powered by data science.

\newpage
\printbibliography

\end{document}
